src/highlight-selected.js

```diff
@@ -28,10 +28,15 @@ window.addEventListener('load', function() {
                 }
             }
         }
-        var el = document.querySelector(query);
-        if (!el) return;
+        var elements = document.querySelectorAll(query);
+        if (elements.length == 0) return;
         textNodes = [];
-        getTextNodesIn(el);
+
+        for (var i = elements.length; i--;) {
```

Comment:
Can you use a [`for..of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) statement here?